checkov/secrets/runner.py

```diff
@@ -286,7 +286,7 @@ def run(
                     code_block=[(secret.line_number, line_text_censored)],
                     file_path=relative_file_path,
                     file_line_range=[secret.line_number, secret.line_number + 1],
-                    resource=secret.secret_hash,
+                    resource=f'{added_commit_hash}:{secret.secret_hash}' if added_commit_hash else secret.secret_hash,
```

Comment:
you can merge the value in line 274 `resource = f'{relative_file_path}:{added_commit_hash}:{secret.secret_hash}' if added_commit_hash else f'{relative_file_path}:{secret.secret_hash}'`

Reply:
resource = f'{added_commit_hash}:{secret.secret_hash}' if added_commit_hash else secret.secret_hash
resource_id = f'{relative_file_path}:{resource}'